activesupport/lib/active_support/dependencies.rb

```diff
@@ -723,6 +723,10 @@ def remove_constant(const) #:nodoc:
       # call from a class or module that were not autoloaded, as in the
       # example above with Object, accessing to that constant must err.
       unless parent.autoload?(to_remove)
+        # Don't try to unload a non-existent constant
+        if !parent.const_defined?(to_remove)
```

Comment:
Can we put this guard clause on one line? `return unless parent.const_defined?(to_remove)`

Reply:
Sure, either way works for me!